processor/src/main/java/org/mapstruct/ap/internal/conversion/Conversions.java

```diff
@@ -193,6 +194,7 @@ public Conversions(TypeFactory typeFactory) {
         register( Currency.class, String.class, new CurrencyToStringConversion() );
 
         register( UUID.class, String.class, new UUIDToStringConversion() );
+        register( URL.class, String.class, new URLToStringConversion() );
```

Comment:
The `URL` is part of the `java.net` module which means that it might not be available. Lets do this in a similar way as it is done for the `java.sql` module. Look at `registerJavaTimeSqlConversions`